crates/aptos-rosetta/src/types/requests.rs

```diff
@@ -56,6 +57,7 @@ pub struct AccountBalanceMetadata {
     pub sequence_number: U64,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub operators: Option<Vec<AccountAddress>>,
+    pub lockup_expiration_time_utc: Option<DateTime<Utc>>,
```

Comment:
Let's leave this as u64 seconds, and let you interpret it on the client side.

These tend to convert time into weird formats.